appd.adoc

```diff
@@ -263,24 +203,23 @@ where
 ----
 C(k) = (1-b)*sinh(a*s(k))/sinh(a) +
        b*[tanh(a*(s(k)+0.5))/(2*tanh(0.5*a)) - 0.5]
-
 ----
 
 where `z(n,k,j,i)` is height (positive upwards) relative to the datum (e.g. mean sea level) at gridpoint `(n,k,j,i)`, `eta(n,j,i)` is the height of the sea surface (positive upwards) relative to the datum at gridpoint `(n,j,i)`, `s(k)` is the dimensionless coordinate at vertical gridpoint `(k)`, and `depth(j,i)` is the distance (a positive value) from the datum to the sea floor at horizontal gridpoint `(j,i)`.
 The constants `a`, `b`, and `depth_c` control the stretching.
 The constants `a` and `b` are dimensionless, and `depth_c` must have units of length.
 
-The format for the **`formula_terms`** attribute is
-----
+The format for the **`formula_terms`** attribute is 
```

Comment:
```suggestion
The format for the `formula_terms` attribute is 
```

(Could batch these up at this point, commit-wise!)

Reply:
As before.